app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,10 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
-    <%= yield %>
+      <% flash.each do |message_type, message| %>
+    <div class="alert alert-<%= message_type %>"><%= message %></div>
+  <% end %>
+   <%= yield %>
+    <%= debug(params) if Rails.env.development? %>
```

Comment:
please check the indenting, have 2 spaces

Reply:
sorry,I am quite confuse about the arrangment for this,too many div

Reply:
will try few time